src/Panel/Field.php

```diff
@@ -0,0 +1,264 @@
+<?php
+
+namespace Kirby\Panel;
+
+use Kirby\Cms\File;
+use Kirby\Cms\Page;
+
+/**
+ * Provides common field prop definitons
+ * for dialogs and other places
+ *
```

Comment:
Tbh this feels like we are adding just another place for field-related code (after `config/field`, `Kirby\Cms` and `Kirby\Form`). Not sure we are doing ourselves a service with that.

Reply:
Yes, definitely. As I wrote in Discord, it's a shortcut because our form code is such a mess. I wish it was completely reusable here, but the complexity of our form implementation did not let me find a solution that was fast enough for the alpha to get this done.

Reply:
Do you think it hurts when we keep this for the alpha and try to find a better solution in the next weeks after the launch?

Reply:
I think that would be fine